components/scenes/main.ts

```diff
@@ -83,10 +79,41 @@ export class MainScene extends Phaser.Scene {
     this.input.keyboard?.on("keydown", (event: KeyboardEvent) => this.handleKey(event), this);
   }
 
+  private createLanes(): void {
+    const lanePositions = [200, 400, 600];
+    const numberRanges = [
+      [10, 39],
+      [40, 69],
+      [70, 99],
+    ];
+
+    for (let i = 0; i < 3; i++) {
+      const x = lanePositions[i];
+      const [min, max] = numberRanges[i];
+      const laneNumber = Phaser.Math.Between(min, max);
+
+      const lane = new Lane(this, x, laneNumber, this.words);
+      this.lanes.push(lane);
+    }
+
+    this.highlightActiveLane();
+  }
+
+  private highlightActiveLane(): void {
+    this.lanes.forEach((lane, index) => {
+      if (index === this.activeLane) {
+        lane.activate();
+      } else {
+        lane.deactivate();
+      }
+    });
+  }
+
   private startGame(): void {
     this.score = 0;
     this.timeRemaining = 10;
```

Comment:
動作確認がしやすいように、最初の残り時間は30秒にしてください。
```suggestion
    this.timeRemaining = 30;
```